packages/client-node/src/client.ts

```diff
@@ -62,7 +62,7 @@ export function createClient(
     }
   }
   const keep_alive = {
-    enabled: config?.keep_alive?.enabled ?? true,
+    enabled: config?.keep_alive?.enabled ?? false,
```

Comment:
1. Why?
2. It's worth mentioning in CHANGELOG.

Reply:
Let's keep it enabled by default now.